DQMOffline/L1Trigger/src/L1TSync_Offline.cc

```diff
@@ -202,14 +202,6 @@ L1TSync_Offline::L1TSync_Offline(const ParameterSet & pset) :
 
   }
 
-  m_outputFile = pset.getUntrackedParameter < std::string > ("outputFile","");
```

Comment:
If this parameter is not used anymore it should also be removed from the config files.